wikipedia-sentences/flows/index.yml

```diff
@@ -0,0 +1,11 @@
+!Flow
+pods:
+  encoder:
+    uses: pods/encode.yml
+    parallel: $JINA_PARALLEL
+    timeout_ready: 600000
+    read_only: true
+  doc_indexer:
+    uses: pods/doc.yml
+    shards: $JINA_SHARDS
+    separated_workspace: true
```

Comment:
separated_workspace is deprecated

Reply:
if we really need to make it work with `shards` we will need to add changes here to make it work properly and gather properly the `matches` , sort them and so on.

Reply:
We don't need shards. New version fixes this issue and `separated_workspace`